lib/webpack.config.js

```diff
@@ -89,6 +89,10 @@ const base = ({ alias = {}, env, tsTranspileOnly, packageJson }) => ({
     //   analyzerMode: 'static',
     //   reportFilename: resolve('target/report.html'),
     // }),
+    new webpack.ContextReplacementPlugin(
+      /graphql-language-service-interface[\\/]dist$/,
```

Comment:
Needed to clean up logs - https://github.com/graphql/graphiql/issues/882

Reply:
fix is coming for this soon!